src/components/ressource/montants.cy.ts

```diff
@@ -0,0 +1,25 @@
+import Montants from "./montants.vue"
+
+describe("<Montants />", () => {
```

Comment:
Ici on peut importer le store grâce à `createTestingPinia` [source](https://github.com/vuejs/pinia/discussions/985#discussioncomment-4063474)

```suggestion
import Montants from "./montants.vue"
import { createTestingPinia } from "@pinia/testing"
import { useStore } from "@/stores/index.js"

describe("<Montants />", () => {
  beforeEach(() => {
    createTestingPinia({
      initialState: {},
      createSpy: cy.spy,
    })
    cy.wrap(useStore()).as("store")
  })
```